File: Cpp/DSA/LinearSearch.cpp
#include <iostream>
using namespace std;

int main() {
    int n, key, found = 0;
    cout<<"Enter the size of an array:";
    cin >> n;

    int arr[n];
    cout<<"Enter the elements to an array:";
    for(int i = 0; i < n; i++)
        cin >> arr[i];
    cout<<"Enter the search element:";

    cin >> key;

    for(int i = 0; i < n; i++) {
        if(arr[i] == key) {
            cout << "Found at index " << i;
            found = 1;
            break;
        }
    }

    if(!found)
        cout << "Not Found";

    return 0;
}

File: Java/DSA/LinearSearch.java
import java.util.*;

public class LinearSearch {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the size of an Array:");
        int n = sc.nextInt();

        int[] arr = new int[n];
        System.out.println("Enter the elements to an Array:");
        for(int i = 0; i < n; i++)
            arr[i] = sc.nextInt();
        System.out.println("Enter the key element:");

        int key = sc.nextInt();
        boolean found = false;

        for(int i = 0; i < n; i++) {
            if(arr[i] == key) {
                System.out.println("Found at index " + i);
                found = true;
                break;
            }
        }

        if(!found)
            System.out.println("Not Found");
    }
}